content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+A low-tech solution using a thermometer, a calendar, and a pencil remains the safest even if it is less attractive.
+
+The FTC sued Kochava on Monday, August 29, 2022 for selling the geolocation information of "hundreds of thousands" of cell phones to anti-abortion groups.  These groups have used geolocation data to identify women who have visited abortion clinics[^10]. The worst future is already now!
+
+#### Shop and e-shops
+
+Some users flag that shops/e-shops that store your buying history may also be a risk. Your purchase history of menstrual hygiene products can also be requested by state police to reveal an illegal abortion. Some propose to desynchronize your purchases of menstrual hygiene products, pay in cash, and cancel your loyalty card (because it is identifying the buyer).
+
+#### Recommendations for users
+
+This abortion law does not only impact the freedom of women, it may also impact their health[^11] [^12]. So it is important to give some advice to enable users to protect themselves.
+
+As usual we can advise to use search engines that consume less data, such as DuckDuckGo and Startpage, use encrypted messaging services such as Signal or ProtonMail, and virtual private networks (VPN) to hide your real location, not browse while identified on some services like Google, clear cookies at browser exit, configure your browser to request web sites not to track you... Use open source app solutions, or cloudless solutions is also a good classic advice. Avoid using social network to discuss or disclose any sensible topics that you may not want to be turned against you later on. Remember that the internet does not forget anything, nothing is really private and something that you said 10 years ago could be an issue in a far future.
+
+#### Last words…
+
+When we consider the global picture, it finally reveals cruelly the real nature of the world we live in today. We accepted because of facility, ignorance, money and comfort to leave a very detailed numeric double of ourselves.
+
+In a healthy society, this should not be a huge problem. On the other hand, in a world where all drifts and evolutions are possible, the digital Human is at the mercy of any legal, neutral or occult force. It has become incredibly complicated, even impossible in these conditions, to protect yourself in the event of abuse.
+
+The other aspect to remember is also that the Digital Humans could experience the quick or progressive suppression of all democratic life. This story therefore reveals the fragility, sometimes wanted by the authorities, of our democracies. Is this electroshock enough for us to take back our digital destiny?
+
+The 5 stages of grief are often said to be shock and denial, anger, bargaining, depression, and finally acceptance. We are, I think, between anger and bargaining. Depression therefore awaits us, the penultimate step towards the real implementation of a set of solutions by the citizens.
```

Comment:
Isn't "I think" in an article not signed by a single individual a little bit strange? The last sentence seems also very strange, like prophetic.

Reply:
Modified.